next-frontend/src/hooks/useQueryTasks.ts

```
@@ -0,0 +1,29 @@
+import { Task } from "@prisma/client"
+import { useQuery } from "@tanstack/react-query"
+import axios, { AxiosError } from "axios"
+import Error from "next/error"
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)